querydsl-docs fails maven build

[ERROR] Failed to execute goal org.jboss.maven.plugins:maven-jdocbook-plugin:2.3.5:resources (default-resources) on project querydsl-docs: Unexpected problem: Unable to locate artifact [org.jboss.jdocbook:jdocbook-core:jar:1.0.4]: Could not find artifact org.jboss.jdocbook:jdocbook-core:jar:1.0.4 in querydsl (http://www.querydsl.com/maven)

--- a/querydsl-docs/pom.xml
+++ b/querydsl-docs/pom.xml
@@ -91,4 +91,27 @@
     </pluginRepository>
   </pluginRepositories>
 
+  <repositories>
+    <repository>
+      <id>jboss</id>
+      <url>https://repository.jboss.org/nexus/content/repositories/releases</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
+    <repository>
+      <id>jboss-thirdparty</id>
+      <url>https://repository.jboss.org/nexus/content/repositories/thirdparty-releases/</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+
 </project>

